src/helpers/agGrid.ts

```diff
@@ -637,9 +653,16 @@ export const getGridIndex = (position: number, gridDistance: number) => {
 
 export const cellRenderer = (params: ICellRendererParams) => {
     const isRowTotal = params.data && params.data.type && params.data.type.rowTotal;
+
+    const isActiveRowTotal =
```

Comment:
is it possible to use valueFormatter instead?
https://www.ag-grid.com/javascript-grid-value-getters/#example-value-getter

Reply:
I don't see how it would work. We use `params.formatValue` below to handle the value formatting. Also, the formatters/getters are defined in `colDefs` which is in another part fo the component